include/vcpkg/base/messages.h

```diff
@@ -2091,5 +2091,168 @@ namespace vcpkg
                     (msg::path),
                     "",
                     "Failed to parse {path}, expected a top-level object.");
-
-}
+    DECLARE_MESSAGE(FailedToParseSerializedManifest,
+                    (msg::path),
+                    "",
+                    "Failed to parse serialized manifest file of {path} Please open an issue at "
+                    "https://github.com/microsoft/vcpkg, with the following output:");
+    DECLARE_MESSAGE(MismatchedSerializedManifestSCF,
+                    (),
+                    "",
+                    "The serialized manifest SCF was different from the original SCF. Please open an issue at "
+                    "https://github.com/microsoft/vcpkg, with the following output:");
+    DECLARE_MESSAGE(PortBugIncludeDirInCMakeHelperPort,
+                    (),
+                    "",
+                    "The folder /include exists in a cmake helper port; this is incorrect, since only cmake "
+                    "files should be installed");
+    DECLARE_MESSAGE(PortBugMissingIncludeDir,
+                    (),
+                    "",
+                    "The folder /include is empty or not present. This indicates the library was not correctly "
+                    "installed.");
+    DECLARE_MESSAGE(PortBugRestrictedHeaderPaths,
+                    (),
+                    "",
+                    "Restricted headers paths are present. These files can prevent the core C++ runtime and "
+                    "other packages from compiling correctly:");
+    DECLARE_MESSAGE(PortBugAllowRestrictedHeaders,
+                    (msg::env_var),
+                    "",
+                    "In exceptional circumstances, this policy can be disabled via {env_var}");
+    DECLARE_MESSAGE(PortBugDuplicateIncludeFiles,
+                    (),
+                    "'{CURRENT_PACKAGES_DIR}' should not be translated.",
+                    "Include files should not be duplicated into the /debug/include directory. If this cannot "
+                    "be disabled in the project cmake, use\n"
+                    "file(REMOVE_RECURSE \"${CURRENT_PACKAGES_DIR}/debug/include\")");
```

Comment:
Does `{CURRENT_PACKAGES_DIR}` need to be `{{CURRENT_PACKAGES_DIR}}` to avoid being interpreted as an fmt replacement? That you had to say `"'{CURRENT_PACKAGES_DIR}' should not be translated."` suggests that it does.

Ditto other occurances.

Reply:
I don't think it needs to be {{...}}. If I do that, it interprets it as a replacement.